src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -748,6 +751,12 @@ public function getPhinxType($sqlType)
             case 'text':
             case 'json':
                 return static::PHINX_TYPE_TEXT;
+            case 'jsonb':
+                return static::PHINX_TYPE_JSONB;
```

Comment:
Would you mind elaborating on your request? JSON and JSONB are two distinct column types in Postgres 9.4 create statements.